### DISCUSSION TITLE
Dimension support

### DISCUSSION THREAD

**Discussion by Colatino** [ognjen-petrovic/js-dxf#88] [Ideas]
I've been trying this library and it's been awesome, thanks for your work on it.

Is it possible to add support for the "Dimension" entity?

**Comment by tarikjabiri** [comment]
Hi

If its ok to switch to another lib, I implemented the dimensions entities you can see the source code in this [line](https://github.com/dxfjs/writer/blob/main/src/DxfWriter.ts#L121), for now there is some entities are not well documented you can see the [docs](https://dxf.vercel.app) or the code source for more informations.

For this lib I'm not sure if this will be possible maybe in the future.

Regards

**Comment by ognjen-petrovic** [comment]
Yes, Tarik is right, please check his lib since I'm occupied with
other projects and Tarik is doing great progress with his fork.

**Comment by Colatino** [comment]
Thanks @tarikjabiri and @ognjen-petrovic, I'll have a look.

Edit: It's working like a charm! Thanks!
